db/schema.rb

```diff
@@ -68,9 +68,6 @@
     t.index ["submitted_by_type", "submitted_by_id"], name: "index_claims_on_submitted_by"
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
This shouldn't have been removed https://github.com/ilyakatz/data-migrate#installation

Reply:
I have no idea how I've manged to remove that 🤯